app/tests/api/api_v1/test_job.py

```diff
@@ -182,7 +182,7 @@ def test_retry_job(
 ) -> None:
     db_job = crud.job.create(db, obj_in=job_create)
     crud.job.update_status(db, db_obj=db_job, status=status)
-    response = client.post(f"{settings.API_V1_STR}/job/{db_job.id}/retry")
+    response = client.post(f"{settings.API_V1_STR}/job/{db_job.id}/retry", json={})
```

Comment:
We should update the tests to reflect that now it's possible to update the job params when requesting a `retry`